taxjar_salestax/models/taxjar_config.py

```diff
@@ -183,10 +183,14 @@ def create_order(
                 'An order without any lines cannot be created in TaxJar')
 
         for item in items:
-            for key in ['unit_price', 'discount', 'sales_tax']:
+            discount_perc = self._get_discount_percentage_from_order_item(item)
```

Comment:
I think you can use `discount_perc = self.env['account.invoice.line'].browse(item['id']).discount` here

Reply:
Or you can add the discount percentage in https://github.com/VanMoof/account-fiscal-rule/blob/add/12.0/taxjar_salestax/taxjar_salestax/models/account_invoice_line.py#L42-L49 and `pop` it from the item here.

Reply:
:+1: thanks!